.travis.yml

```diff
@@ -0,0 +1,48 @@
+---
+os: linux
```

Comment:
Hmm, why go back to Travis? Containers don't work w/ GitHub runners?

Reply:
No the containers won't work with github runners because github doesn't have a hole in the vpn to reach our internal artifactory and download the docker images.

Reply:
Gotcha, thanks.